llvm/test/Other/debugcounter-slsr.ll

```diff
@@ -0,0 +1,26 @@
+; REQUIRES: asserts
+; RUN: opt -passes=slsr,gvn -S -debug-counter=slsr-counter=1  < %s | FileCheck %s

```

Comment:
Do we need to run GVN (Global Value Numbering) here? It'd be nicer if you could eliminate the need to run it.

Reply:
No, we don't I have removed it.